src/engine/tools/engine-suite/src/engine_test/cmds/run.py

```diff
@@ -25,6 +26,9 @@ def configure(self, subparsers):
         parser_run.add_argument('-i', '--agent-id', help=f'Agent ID for event filling',
                                 type=str, default=DEFAULT_AGENT_ID, dest='agent_id')
 
+        parser_run.add_argument('--api-socket', help=f'Socket to connect to the API',
```

Comment:
Maybe we should add the short argument, like -S or -A, to follow the style of the rest of the arguments.